pkg/controller/networkpolicy/tier.go

```diff
@@ -158,3 +177,31 @@ func (n *NetworkPolicyController) initTier(t *secv1alpha1.Tier) {
 		return
 	}
 }
+
+// updateTier attempts to update Tiers using an
+// exponential backoff period from 1 to max of 8secs.
+func (n *NetworkPolicyController) updateTier(t *secv1alpha1.Tier) {
+	var err error
+	const maxBackoffTime = 8 * time.Second
+	backoff := 1 * time.Second
+	retryAttempt := 1
+	for {
+		klog.V(2).Infof("Updating %s Tier", t.Name)
+		_, err = n.crdClient.SecurityV1alpha1().Tiers().Update(context.TODO(), t, metav1.UpdateOptions{})
+		// Attempt to update Tier after a backoff.
+		if err != nil {
+			klog.Warningf("Failed to update %s Tier on init: %v. Retry attempt: %d", t.Name, err, retryAttempt)
+			// Tier update may fail because antrea APIService is not yet ready
+			// to accept requests for validation. Retry fixed number of times
+			// not exceeding 2 * 5 = 10s.
```

Comment:
what does the comment 2*5 mean? the maxBackoffTime is 8 seconds in code.

Reply:
old comment .. new code... old comment copy paste from above.. updated in both places